plugins/mysql/mysql_audit

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/python
+# -*- python -*-
+
+"""
+=head1 INTRODUCTION
+
+Plugin to monitor the MySQL audit log connection count
+
+=head1 INSTALLATION
+
+Place in /etc/munin/plugins/ (or link it there using ln -s)
+
+=head1 CONFIGURATION
+
+Add this to your /etc/munin/plugin-conf.d/munin-node:
+
+=over 2
+
+      [mysql_audit]
+      user mysql
+      period 3
```

Comment:
I think, you omitted the `env.` prefix here.

Reply:
Oops, sorry again.